src/app/models/types/translations.ts

```diff
@@ -285,3 +278,12 @@ export type UgcTranslations = {
   closedHeading: string;
   closedDescription: string;
 };
+
+export interface LiteSiteTranslations {
```

Comment:
I always wonder what the difference between a `type` and an `interface` is? According to the doc they are pretty much interechangable - https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#differences-between-type-aliases-and-interfaces (my personal preference is `interface` for something like this, but we tend to use `type` more across Simorgh 🤷‍♀️ )

Reply:
Yeah I've always tended to use interfaces for object-like types, but under the hood there's not much difference